daemon/algod/api/spec/v1/model.go

```diff
@@ -223,6 +224,35 @@ type PaymentTransactionType struct {
 	CloseRewards uint64 `json:"closerewards"`
 }
 
+// KeyregTransactionType contains the additional fields for a keyreg Transaction
+// swagger:model KeyregTransactionType
+type KeyregTransactionType struct {
+	// VotePK is the participation public key used in key registration transactions
+	//
+	// required: false
+	VotePK bytes `json:"votekey,omitempty"`
```

Comment:
We shouldn't use `omitempty`, there's been a push to remove it from our API.

Reply:
@Vervious Just for the fields or also in the payment and keyreg options structs too?

Reply:
Good point. Seems like the `Transaction.payment` and `Transaction.keyreg` fields are special cases. Let's keep `omit empty` there for now, I think?

Reply:
See https://github.com/algorand/go-algorand/pull/161

Reply:
Perfect. Doing that.